common/txmgr/broadcaster.go

```diff
@@ -64,6 +64,26 @@ var ErrTxRemoved = errors.New("tx removed")
 
 type ProcessUnstartedTxs[ADDR types.Hashable] func(ctx context.Context, fromAddress ADDR) (retryable bool, err error)
 
+type saveTryAgainAttemptParams[
```

Comment:
static analysis complained function signature too long so I created a struct for it

Reply:
Please remove this